From b17ba818693a7d65bd66978a54b6250ddb14143d Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Wed, 6 Mar 2019 11:12:54 +0100
Subject: [PATCH 024/386] arm64: dts: cn9131: use correct address for phy 3310

The 3310 PHY address is 8 and not 4.

Change-Id: Ia8f8cf7c36f4c7e3c0212c18db95e80d1c98f588
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/5334
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9131-db.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
index a9b226d44afb..f8f5cd04b79d 100644
--- a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
@@ -82,7 +82,7 @@
 &cp1_eth1 {
 	status = "disabled";
 	phy-mode = "10gbase-kr";
-	phy = <&phy4>;
+	phy = <&phy8>;
 	/* TODO: regarding to the spec comphy2 can be used with port 0 only */
 	phys = <&cp1_comphy2 1>;
 };
@@ -198,9 +198,9 @@
 
 &cp1_xmdio {
 	status = "okay";
-	phy4: ethernet-phy@4 {
+	phy8: ethernet-phy@8 {
 		compatible = "ethernet-phy-ieee802.3-c45";
-		reg = <4>;
+		reg = <8>;
 	};
 };
 
-- 
2.17.1

